.github/PULL_REQUEST_TEMPLATE/maintainer_nomination.md

```diff
@@ -0,0 +1,17 @@
+# Nomination for New Maintainers
```

Comment:
wondering if this should be amended to apply to one maintainer at a time?

Reply:
I'll update it